### ISSUE TITLE
flat package when built on windows

### ISSUE THREAD

**Issue by BottlecapDave** [testfairy/testfairy-unity-asset-packager#2]
Hi.

When I use your tool locally on a mac it packages up my unity project wonderfully - Thank you. 

However, when I attempt to use it on Windows (our build box) and import the created package, it seems to not be packaging directories correctly.

e.g.
```
-> Project
-> Project/Scripts
-> Project/Scripts/ScriptA.cs
...
```
instead of

```
-> Project
--> Project/Scripts
---> Project/Scripts/ScriptA.cs
...
```

I don't supposed you know why this would be?

The end result is that I am unable to import the package as it complains that the directories don't exist

**Comment by vijaysharm** [comment]
Hey @BottlecapDave! Thanks for reaching out. Always happy to hear someone else has found this tool is helpful!

This issue isn't something we've come across yet, so thank you for bringing it to our attention. We'll have to take a look at it and get back to you.

**Comment by vijaysharm** [comment]
Hi @BottlecapDave, I just published version 1.1.0 which should fix your issue. It looks like we were not converting to the correct Posix path expected by Unity when run from Windows.

I'm going to close this issue. If you continue to have problems, feel free to reopen this issue.

Cheers!
